core/src/main/java/org/testcontainers/containers/BaseConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+
+import java.util.List;
+
+public interface BaseConfiguration {
```

Comment:
Is this interface supposed to be part of this PR? Where is it used?

Reply:
it was leaked. will remote it